client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -363,6 +363,37 @@ <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_configuration"><
                                         <span data-ng-bind="hosting.filer"></span>
                                     </div>
                                 </li>
+                                <li class="oui-tile__item" data-ng-if="hosting.isCloudWeb">
+                                    <div class="oui-tile__definition">
+                                        <div class="d-flex w-100">
+                                            <div>
+                                                <strong class="d-block"
+                                                        data-i18n-static="hosting_dashboard_service_default_runtime"></strong>
+                                                <span data-ng-bind="ctrl.defaultRuntime.name"
+                                                      data-ng-if="ctrl.defaultRuntime"></span>
+                                                <div class="d-inline ml-2" data-ng-if="!ctrl.defaultRuntime">
+                                                    <oui-spinner data-size="s"></oui-spinner>
+                                                </div>
+                                            </div>
+                                            <div class="mb-auto ml-auto">
+                                                <button class="oui-button oui-button_dropdown mb-0" type="button" aria-haspopup="true" aria-expanded="false"
```

Comment:
Didn't know about `aria-haspopup`:+1:

Reply:
`aria-expanded`'s value should change wether the menu is expanded or not, doesn't it ?